build.sbt

```diff
@@ -84,6 +84,11 @@ def hasNewParser(versionString: String) = versionString match {
   case _ => false
 }
 
+def hasNewPlugin(versionString: String) = versionString match {
+  case HasScalaVersion(2, 10, _) => false
```

Comment:
Do we need the Scala 2.10 support still? Otherwise I'm inclined to drop this.

Reply:
@larsrh Do you want me to drop 2.10 build as part of this PR? The `case` is required to compile on 2.10 but is not required otherwise.

Reply:
That would be great!

Reply:
@larsrh Done in last 2 commits